spotify.go

```diff
@@ -40,14 +40,19 @@ const (
 	rateLimitExceededStatusCode = 429
 )
 
+// ErrNoMorePages is the error returned when response header has a 'Retry-After'
+// duration longer then the configuerd max.
```

Comment:
oops one more I missed the first time

```suggestion
// duration longer then the configured max.
```